### DISCUSSION TITLE
Accidental 'chezmoi update' overwrote files... is there a way to undo this change?

### DISCUSSION THREAD

**Discussion by sweetbbak** [twpayne/chezmoi#2662] [Q&A] [answered]
I accidently did "chezmoi update" which ended up deleting/changing quite a few files, is there any straightforward way that I can restore these files? If I do chezmoi diff I can see one file that it changed but not all of them. I dont want to shoot around in the dark here and end up doing more damage.

**Comment by twpayne** [comment, marked as answer]
You'll have to restore your files either from your git history or from your backups.

**Reply by sweetbbak** [reply]
damn Im beat lol my .bak files were overwritten as well. I guess this is a learning experience for me. Thanks for the help.
